README.md

```diff
@@ -22,6 +22,15 @@ Read the full Parse Server guide here: https://github.com/ParsePlatform/parse-se
 * You now have a database named "dev" that contains your Parse data
 * Install ngrok and you can test with devices
 
+### These scripts can help you to develop your app for Parse Server
+
+* `npm run watch` will start your Parse Server and restart if you make any changes.
+* `npm run lint` will check the linting of your cloud code, tests and `index.js`, as defined in `.eslintrc.json`.
+* `npm run lint-fix` will attempt fix the linting of your cloud code, tests and `index.js`.
+* `npm run prettier` will help improve the formatting and layout of your cloud code, tests and `index.js`, as defined in `.prettierrc`.
+* `npm run test` will run any tests that are written in `/spec`.
+* `npm run coverage` will run tests and check coverage. Output is available in the `/coverage` folder.
+
 ### Getting Started With Heroku + mLab Development
```

Comment:
Please keep headings short and longer text in the paragraph.

Reply:
I'm not quite sure what you mean. Should it be:

### npm run watch

This will start your Parse Server and restart if you make any changes.

Reply:
I mean that the README currently is hard to read. It needs a ToC. Once you add a ToC (preferably auto-generated), then you'll see that the headlines are repetitive and too long.

There are currently multiple headlines "Getting Started With xxx" instead of a main headline "Getting Started" and sub-headlines "Heroku", "Google Cloud Engine", etc.
Beside that, the capitalization is also quite uncommon, the "with" in "Getting Started With xxx" should not be capitalized.

Reply:
Ok, I think that should be ok now?

Reply:
Yes, that looks better. Would you mind if I jump in and make a commit?

Reply:
Of course! I'll share with you now

Reply:
Thanks, please take a look and add any improvements you may see. Otherwise this PR looks good to me.

Reply:
Thanks for your assistance 😊. It might be worth making sure all the deploy buttons still work (I think AWS doesn't work anymore), but these are good improvement for this PR.